src/CarbonAware.Aggregators/test/ForecastAggregatorTests.cs

```diff
@@ -0,0 +1,277 @@
+﻿using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.Aggregators.Forecast;
+using CarbonAware.Aggregators.Tests;
+using CarbonAware.Interfaces;
+using CarbonAware.Model;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CarbonAware.Aggregator.Tests;
+public class ForecastAggregatorTests
+{
+
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
Did this disable survive through the refactor? Would it just be easier to declare a constructor / mark a field nullable?

Reply:
There were no changes to the functionality. The code was pulled from CarbonAwareAggregator to this new class.

Reply:
We're opting to leave this as-is for now, but it may be worth adding a clean-up story to actually disable this warning for all tests in the `.csproj` files.   Using `[Setup]` rather than constructors is a typical pattern and the runtime safety this warning is concerned with is not a big concern with unit tests.